index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Header Section (Toolbar)-->
+    <header class="header backgrnd-clr">
+      <nav class="navbar">
+        <a href="index.html"><img src="images/icons/Welcome.png" alt="logo" /></a>
+        <img src="images/icons/Union.png" alt="hamburger-menu" />
+      </nav>
+    </header>
+    <!--Main content-->
+    <main class="main-content">
+      <!--Headline Section-->
+      <article class="head-line backgrnd-clr">
+        <!--Greeting text-->
```

Comment:
- [ OPTION] kindly remove unneeded comment inside your HTML file to make look clear and more readable